build/mvn

```diff
@@ -38,7 +38,7 @@ install_app() {
   # setup `curl` and `wget` silent options if we're running on Jenkins
   local curl_opts="-L"
   local wget_opts=""
-  if [ -n "$AMPLAB_JENKINS" ]; then
+  if [ -n "$AMPLAB_JENKINS" -o -n "$K8S_JENKINS" ]; then
```

Comment:
Maybe we should stick to AMPLAB_JENKINS so it would be easier getting merged in the future?